src/hooks/requests/useTranslationMemories.ts

```diff
@@ -386,3 +387,41 @@ export const useToggleTmWritable = () => {
     isLoading,
   }
 }
+
+export const useCreateEmptyTm = ({
+  subProjectId,
+}: {
+  subProjectId?: string
+}) => {
+  const queryClient = useQueryClient()
+  const { mutateAsync: createEmptyTm, isLoading } = useMutation({
+    mutationKey: ['subProject-tm-keys', subProjectId],
+    mutationFn: () => apiClient.post(`${endpoints.TM_KEYS}/${subProjectId}`),
+    onSuccess: ({ data }) => {
```

Comment:
**[blocker]**

Can't merge right now, "undefined" will be added to the list of translationMemories, which will crash the app in TranslationMemorySection. This is waiting for BE to add the missing data to the response ?